jenkins-integration/beaker/install/foss/80_run_agent_on_master.rb

```diff
@@ -5,5 +5,6 @@
   # ahead of the Gatling run avoids collisions in the priming that can happen
   # when multiple agent runs happen at the same time.  See PUP-6651 for more
   # information.
-  on(master, 'puppet agent -t --server `hostname`')
+  fqdn = fact_on(master, 'fqdn')
+  on(master, "puppet agent -t --server #{fqdn}")
```

Comment:
I think as long as `master` will spit out the fqdn that should work fine. I added this because puppet runs were failing, since `--server ``hostname`` ` was only returning the hostname and there was a name mismatch. Could be that I just misconfigured something though. 

Looks like `master` comes from the hosts.yaml, which comes from the SUT_HOST variable when you kick off the job. So that's probably good